### DISCUSSION TITLE
How do you get hot reloading to work for Laravel 11 sites when changing files?

### DISCUSSION THREAD

**Discussion by octoxan** [beyondcode/herd-community#456] [Q&A] [answered]
I just installed Herd fresh on this laptop, then added a new Laravel 11 site with "laravel new example". During install I selected Jetstream and Livewire.

The site loads fine in the browser at https://example.test (enabled the SSL). When I run `npm run dev` it says it's running on https://example.test:5173 and that it's using Herds certificate. That page of course just shows the Vite info page saying it's running.

How do you view the live reloading version of the site? 

https://example.test does not live reload, and I wouldn't really expect that one to. 

I tried running `art serve` before the `npm run dev` and that let's me view the site at `127.0.0.1:8000` but it doesn't reload when CSS/JS/Blade files are changed, even though the terminal makes a mention of every file I am changing. 

Is there anything special that needs done with the vite.config?

I haven't changed anything from a normal Jetstream install yet.

I'm on the free version of Herd btw, and have tried viewing both the .test domain and the artisan serve localhost version of the domain in Chrome, FF, and Safari.

**Comment by octoxan** [comment, marked as answer]
Turns out the welcome.blade.php doesn't have the scripts required for hot reloading and that's where I was testing it on.

**Reply by bn-l** [reply]
wait what are those scripts?
